resource-sharing-partners/src/test/java/test/utils/MerknaderGenerator.java

```diff
@@ -0,0 +1,152 @@
+package test.utils;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomElement;
+import static no.unit.nva.testutils.RandomDataGenerator.randomInteger;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import java.util.Collection;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import no.nb.basebibliotek.generated.Adgang;
+import no.nb.basebibliotek.generated.BetalMrk;
+import no.nb.basebibliotek.generated.FjGen;
+import no.nb.basebibliotek.generated.FjMrk;
+import no.nb.basebibliotek.generated.FjSpes;
+import no.nb.basebibliotek.generated.IndSyst;
+import no.nb.basebibliotek.generated.Klass;
+import no.nb.basebibliotek.generated.KlassNoter;
+import no.nb.basebibliotek.generated.KopGeb;
+import no.nb.basebibliotek.generated.KopMrk;
+import no.nb.basebibliotek.generated.Merknader;
+import no.nb.basebibliotek.generated.Omtale;
+import no.nb.basebibliotek.generated.SpesSaml;
+import no.unit.nva.language.LanguageConstants;
+
+public class MerknaderGenerator {
+
```

Comment:
Kunne vært en privat konstruktør her for å hindre instansiering av klassen.

Reply:
fikset